Dockerfile

```diff
@@ -32,9 +30,10 @@ ARG FLUTTER_BUILD_BRANCH=stable
 ENV FLUTTER_BUILD_BRANCH=$FLUTTER_BUILD_BRANCH
 ENV FLUTTER_ROOT=flutter
 ENV FLUTTER_BIN=flutter/bin
-ENV PATH="/app/flutter/bin:$PATH"
+ENV PATH="/flutter/bin:$PATH"
```

Comment:
I used to remove this, and found out some commands won't be able to find the `flutter`. We should find out how can we rely on `FLUTTER_ROOT`.

Reply:
I also had that issue but I think specifying it as a volume in this PR fixed those cases for me. 

Could you try out these changes?